tests/integration/test_mask_indices.py

```diff
@@ -19,25 +19,94 @@
 import cunumeric as num
 
 KS = [0, -1, 1, -2, 2]
+FUNCTIONS = ["tril", "triu"]
```

Comment:
prefer tuples unless mutability is an explicit need
```suggestion
KS = (0, -1, 1, -2, 2)
FUNCTIONS = ("tril", "triu")
```
(here, and also several places below)

Reply:
You are right. Tuple is better than list if the case is immutable. Fixed.